src/tests/generatekey.cpp

```diff
@@ -821,7 +832,7 @@ TEST_F(rnp_tests, generatekeyECDSA_explicitlySetBiggerThanNeededDigest_ShouldSuc
     assert_true(rnp_cfg_setstr(&ops, CFG_HASH, "SHA512"));
     assert_true(rnp_cfg_setint(&ops, CFG_S2K_ITER, 1));
 
-    assert_true(rnp_cfg_setstr(&ops, CFG_USERID, "expert_large_digest"));
+    assert_true(rnp_cfg_addstr(&ops, CFG_USERID, "expert_large_digest"));
```

Comment:
We should also have negative test case - where two user ids are specified and key generation fails.